### ISSUE TITLE
publish the package to pypi repository

### ISSUE THREAD

**Issue by tngTUDOR** [BONSAMURAIS/ystafdb#9]
Once we publish this to pypi, `ystafdb` will be easier to install (without clonning the repo) and use.

**Comment by tngTUDOR** [comment]
curren state yields and error when trying to upload using twine:

```
HTTPError: 400 Bad Request from https://test.pypi.org/legacy/
The description failed to render in the default format of reStructuredText. See https://test.pypi.org/help/#description-content-type for more information.
```

**Comment by tngTUDOR** [comment]
The formatting issue was a problem with the license (fixed in: 3900b09)

**Comment by tngTUDOR** [comment]
the package can now be seen at:
[https://pypi.org/project/ystafdb/](https://pypi.org/project/ystafdb/)
